### DISCUSSION TITLE
How to avoid the function signature covering the entire screen

### DISCUSSION THREAD

**Discussion by YuFei29** [NvChad/NvChad#2325] [Q&A] [answered]
before I `enter`

![图片](https://github.com/NvChad/NvChad/assets/125109356/718378bd-b995-49dc-b7d4-340ee9765480)
after I `enter`
![Screenshot_20230822_204028](https://github.com/NvChad/NvChad/assets/125109356/7bd370f7-f258-4db4-80db-e7936a15873c)

Thanks a lot

**Comment by siduck** [comment, marked as answer]
duplicate of https://github.com/NvChad/ui/issues/95

**Reply by YuFei29** [reply]
may be  [https://github.com/ray-x/lsp_signature.nvim](https://github.com/ray-x/lsp_signature.nvim) can solve this problem? 

they have option like 
 ```
  max_height = 12, -- max height of signature floating_window
  max_width = 80, -- max_width of signature floating_window

  wrap = true, -- allow doc/signature text wrap inside floating_window, useful if your lsp return doc/sig is too long
```

![Screenshot_20230822_210421](https://github.com/NvChad/NvChad/assets/125109356/98f81870-99ee-4d12-8883-d5f60fdbea07)

**Reply by siduck** [reply]
@YuFei29 didnt know! anyways PRs are always welcomed, i'm a bit busy these days so i'm able to work on my opensource stuff on weekends only sadly.
